FRPs/FRP-6.md

```diff
@@ -0,0 +1,52 @@
+# FRP-6: Literature review of auction mechanism that fit in with Flashbots.
+
+
+**MEV Fellow:** @sbaks0820
+</br> **Status:** Draft
+</br> **Updated:** 24.11.2020
+
+### Abstract
+The puspose of this isssue is to investigate and survey the existing literature of auction mechanism design to motivate the design of the block space and transaction ordering auctions enabled by MEV-GETH.
+An MEV-GETH auction is different from a traditional auction in that users submit and bid on tx sequences that are, potentially, non-overlapping.
+The current implementation of MEV-GETH accepts only a single transacton sequence into a block, however, a more sophisticated auction mechanism might permit multiple "bundles" with interaction between winning, non-overlapping bundles.
+Therefore carefull consideration is required to design a good auction mechanism for MEV-GETH.
+In addition to surveying existing literature, it is impotant to survey auction designs in existing crypto projects so understand the practical requirements of a decentralized auction such as privacy, security, auditability, and efficiency.
+
+### Defining the Research Question
+
+The high-level research question is: What is a good auction mechanism for MEV-GETH auctions? 
+
+The first step in answering this broad question is first surveying and evaluating the existing literature and practice of auction mechanism design in a decentralized setting.
+In order to tackle this objects is to answer the following questions:
+* What assumptions can be made about the participants and their capabilities (includes a potentially malicious miner)?
+* Does expanding the auction format (from a single winning bundle to potentially multipe winning bundles) make auction design much more difficult?
+* How do current implementation auctions in crypto deal with issues such as efficiency, security, privacy, and auditability?
+* In the field of combinatorial auctions: are there existing theoretical auctions that imply constraints on our setting in terms of computability of optimal strategy?
+* In the area of computational complexity: are there computationally difficult auctions with good enough heuristic, or sub-optimal, solutions?
+* How does auction theory reason about potentially colludin parties?
+* What are the computation bounds on miners in determining auction winner(s)?
+* How can we scale auctions to many bots/participants?
+* What are the tradeoffs involved in implementing such an auction in L2?
```

Comment:
Other questions is how crypto models must differ from existing literature.

e.g. impact of easy collusion online on forming cooperative strategies, do we need cooperative game theory and does current auction literature fall short?  I'd imagine ad auctions also share a similar adversarial model; perhaps worth collecting industry anecdotes / research on how these scale?  I know they are now large and very adversarial.  I know Dan Moroz has thought about this a lot, definitely worth getting his thoughts on questions here.

Reply:
Resolved in b56d23e

Reply:
I had something about gathering anecdotal insight on crypto models for auctions in the Methodology section. I think that'll yield more than just surveying literature.